README.md

```diff
@@ -1 +1,39 @@
-# Hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-world
+
+> The goal here is to master all of the tools and best practices we learned about github flow
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
```

Comment:
- [ ] [OPTIONAL] Kindly change the list of technologies that you used to build this project here. (HTML, CSS, linters, ...etc)